settings/src/components/totp.js

```diff
@@ -78,44 +116,39 @@ function Setup() {
 		}
 	} );
 
+	const handleClick = useCallback( () => setIsNextClick( false ), [] );
+
 	return (
 		<>
-			<SetupProgressBar step="totp" />
-
-			<p>
-				Two-Factor Authentication adds an extra layer of security to your account.
-				Once enabled, logging in to WordPress.org will require you to enter a unique passcode
-				generated by an app on your mobile device, in addition to your username and password.
-			</p>
-
-			<p>
-				Not sure what this screen means? You may need to download Authy or Google Authenticator for your phone.
-				{/* todo add links to those. maybe pick different ones? or link to an article/video where they can learn more */ }
-			</p>
-
-			<SetupMethod
-				setupMethod={ setupMethod }
-				setSetupMethod={ setSetupMethod }
-				qrCodeUrl={ qrCodeUrl }
-				secretKey={ secretKey }
-			/>
-
-			<p>Then enter the six digit code provided by the app:</p>
-
-			<SetupForm
-				handleEnable={ handleEnable }
-				verifyCode={ verifyCode }
-				setVerifyCode={ setVerifyCode }
-				qrCodeUrl={ qrCodeUrl }
-				secretKey={ secretKey }
-			/>
-
-			{ error &&
-				<Notice status="error" isDismissible={ false }>
-					<Icon icon={ cancelCircleFilled } />
-					{ error }
-				</Notice>
-			}
+			<SetupProgressBar step="totp-setup" />
+
+			<Flex expanded={false} direction='column' align="top" justify="top" gap="16px" className="wporg-2fa__totp_setup-container">
+				<SetupMethod
+					setupMethod={ setupMethod }
+					setSetupMethod={ setSetupMethod }
+					qrCodeUrl={ qrCodeUrl }
+					secretKey={ secretKey }
+				/>
+
+				<strong>Enter the six digit code provided by the app</strong>
+
+				<SetupForm
+					handleEnable={ handleEnable }
+					qrCodeUrl={ qrCodeUrl }
+					secretKey={ secretKey }
+					inputs={inputs}
+					setInputs={setInputs}
+				/>
+
+				<Button variant="link" onClick={ handleClick }>Previous</Button>
```

Comment:
How does this differ from the "cancel" button under the 6 inputs?

Reply:
Oh, right! Originally, when the cancel was clicked, it would return to the account status screen. Replacing `previous` for that would make the look better. Thanks!

Reply:
I found that it makes sense to me to use the `Cancel` button to clear all 6 inputs and keep the `previous` button.
Commit added: [Let Cancel clear all the inputs](https://github.com/WordPress/wporg-two-factor/pull/100/commits/c4da34ce66f11e9b5fff6c0c8610aa7cb7fa8ac2)